### PR TITLE
fix(pci.ips): use common order flow for pci ips

### PR DESCRIPTION
**Pull request by lizardK**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Use common order flow when ordering failover ips for PCI 

Signed-off-by: Cyril Biencourt <cyril.biencourt@ovhcloud.com>

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10126,16 +10126,11 @@ flat-cache@^3.0.4:
     flatted "^3.1.0"
     rimraf "^3.0.2"
 
-flatpickr@^4.6.3:
+flatpickr@4.6.9, flatpickr@^4.6.3, flatpickr@~4.5.2:
```
Comment (zakariasahmane) [inline yarn.lock:10129]: I don't see any changes in any `package.json`, how come the yarn.lock is different?

File: packages/manager/modules/pci/src/projects/project/failover-ips/onboarding/order/order.routing.js [hunk 2]
```diff
@@ -1,70 +1,43 @@
 import filter from 'lodash/filter';
-import map from 'lodash/map';
+import get from 'lodash/get';
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/failover-ips/onboarding/order/order.routing.js:2]: Cannot be replaced with the optional chaining operator ?

File: packages/manager/modules/pci/src/projects/project/failover-ips/onboarding/order/order.routing.js [hunk 3]
```diff
@@ -1,70 +1,43 @@
...
-                  (id) =>
-                    OvhApiCloudProjectRegion.v6().get({
-                      serviceName: projectId,
-                      id,
-                    }).$promise,
+    },
+    resolve: {
+      breadcrumb: /* @ngInject */ () => null,
+      /* @ngInject */
+      products: (OvhApiOrderCatalogFormatted) =>
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/project/failover-ips/onboarding/order/order.routing.js:22]:
```suggestion
      products: /* @ngInject */ (OvhApiOrderCatalogFormatted) =>
```
shouldn't be this way ?

**Review by antleblanc — DISMISSED** [review]

File: yarn.lock (hunk 1 above)
Comment (antleblanc) [reply yarn.lock:10129]: Rebase on `master` will fix it.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**